README.md

```diff
@@ -16,8 +16,7 @@ You can run the unit tests by simply running  the command: `go test -cover -race
 
 #### Development and Testing
 
-To try the  in test mode (using [goaws](https://github.com/p4tin/goaws))
+To try the  in test mode (using [localstack](https://hub.docker.com/r/localstack/localstack))
```

Comment:
goaws is not compatible with newer AWS SDK - I needed to replace it

Reply:
I don't know why but I remember experiencing strange behaviours with localstack.

Reply:
unfortunetly goaws doesn't work with never AWS SDK: https://github.com/Admiral-Piett/goaws/issues/279

but it looks lighter than localstack so we can potentially go back